src/components/common/TmSessionExplore.vue

```diff
@@ -127,6 +134,18 @@ export default {
         return false
       }
     },
+    isNotAnyOtherCosmosAddress(param) {
```

Comment:
instead of blacklisting prefixes, how about checking if the prefix is whitelisted => a `bech32_prefix` in any network

Reply:
OK, I think I just implemented it like you had in mind